.eslintrc.json

```diff
@@ -234,7 +264,13 @@
                     "off"
                 ],
                 "no-await-in-loop": [
-                    "off"
+                    "error"
```

Comment:
Too limiting. There're good examples when one would want a sequential execution of promises instead of parallel one for performance reasons, for example. No await in `while` or `for` doesn't bring any advantage except for very few very obvious cases which I hope we're all capable of avoiding.

Reply:
I would say forcing tasks in parallel is a good idea and sequential can be treated as an exception, would make you think twice do you really need it as a sequential process or not

Reply:
I am 50/50. Sequential execution can be necessary sometimes. But it is vulnerable to misuse for example sequential db calls (I think this is what Artem means while saying very obvious cases).

Reply:
yes, but eslint idea is to prevent possible errors, and using this rule would prevent of processing promises in sequence unintentionally

Reply:
I think our CTO should fire the person who commit that effective immediately :D

Reply:
everyone can make mistakes :)

Reply:
So @iartem this is the last one, will you budge on this or you stand your position? :D

Reply:
I just don't like meaningless limitations, but I won't be too loud about this one :D